### ISSUE TITLE
For SQS can we find a way to access a queue without having to list queues first?

### ISSUE THREAD

**Issue by brunchey** [agorapulse/grails-aws-sdk#60]
Hello, I am wondering if it would be possible to modify the code to avoid listing the queues first before accessing the specified queue?  Right now I have to provide access to all queues for a given region so that the list command can be ran.  Is this by design?

**Comment by musketyr** [comment]
We currently load all queues to get the URL of the queue for given name. We'll take a look to figure out if there is a way how we construct the URL without asking for all queues before.

**Comment by benorama** [comment]
Hi @brunchey ,

A quick workaround proposal: we could support a `queueUrl` or `queueName` as a parameter for all methods (we would just detect `https:` in the parameter).
So you would just have to pass the correct queue url, since you know it already.

Exemple:
```
// Instead of
amazonSQSService.sendMessage(queueName, messageBody)

// You would do
amazonSQSService.sendMessage(queueUrl, messageBody)
```

What do you think?

**Comment by brunchey** [comment]
Let me check with my team.  I like the idea.

**Comment by brunchey** [comment]
Yes, this would be great!

**Comment by benorama** [comment]
Available in latest release: `2.1.18`
Let us know how it goes!

**Comment by brunchey** [comment]
Thanks!
